options/options.cpp

```diff
@@ -271,6 +272,13 @@ const option::Descriptor usage[] = {
     Arg::None,
     "  --cegp-abs-vals \tabstract values in ceg-prophecy-arrays (only "
     "supported for IC3IA)" },
+  { CEGP_ABS_VALS_CUTOFF,
+    0,
+    "",
+    "cegp-abs-vals-cutoff",
+    Arg::Numeric,
+    "  --cegp-abs-vals-cutoff \tcutoff value for what to abstract (must be "
```

Comment:
shouldn't we also print the default value? (which I think is 100)

Reply:
Good point! Made change with c1f5d53522acd2ab1d8181860a26881ff50edc83.